MovieQuiz/Presentation/MovieQuizPresenter.swift

```diff
@@ -0,0 +1,144 @@
+//
+//  MovieQuizPresenter.swift
+//  MovieQuiz
+//
+//  Created by Павел Афанасьев on 06.03.2023.
+//
+
+import UIKit
+
+final class MovieQuizPresenter: QuestionFactoryDelegate {
+    private let statisticService: StatisticService!
+    private var questionFactory: QuestionFactoryProtocol?
+    private weak var viewController: MovieQuizViewController?
+    
+    private let questionsAmount: Int = 10
+    private var currentQuestionIndex: Int = 0
+    private var currentQuestion: QuizQuestion?
+    private var correctAnswers: Int = 0
+    
+    init(viewController: MovieQuizViewControllerProtocol){
```

Comment:
Будь внимателен, после круглой скобки и перез фигурной всегда есть пробел:
```suggestion
    init(viewController: MovieQuizViewControllerProtocol) {
```